docs/dt.md

```diff
@@ -0,0 +1,181 @@
+            "SubjectName": "*.g.doubleclick.net",
+            "ValidFrom": "2018-12-19 08:17:00",
+            "ValidTo": "2019-03-13 08:17:00",
+            "Issuer": "Google Internet Authority G3"
+        }, {
+            "SubjectName": "*.apis.google.com",
+            "ValidFrom": "2018-12-19 08:16:00",
+            "ValidTo": "2019-03-13 08:16:00",
+            "Issuer": "Google Internet Authority G3"
+        }],
+        "ASN": "AS15169"
+    },
+    "MaxMind": {
+        "Address": "8.8.8.8",
+        "ISP": "Google",
+        "UserType": "business",
+        "Organization": "Google LLC",
+        "ISO_Code": "US",
+        "Geo": {
+            "Location": "37.751, -97.822",
+            "Country": "United States",
+            "Continent": "North America",
+            "Accuracy": 1000
+        },
+        "ASN": 15169,
+        "RegisteredCountry": "United States"
+    }
+}
+```
+
+## Nested Values
+DT can go deep. Deep like cosmic brownies and Jimi Hendrix at 2am deep. But in its essence, we are dealing with a dictionary of key values. Let's look at the following example.
+
+Using the context example noted above, these are the various ways that DT can be used to access nested keys.
+
+| Example | Result |
+| --- | --- |
+| ```${MaxMind.Geo.Continent}``` | North America |
+| ```${IP.Hostname}``` | google-public-dns-a.google.com |
+| ```${IP.RecordedFuture.FirstSeen}``` |  2010-04-27T12:46:51.000Z |
+
+
+## Dealing with Arrays
+We access array values just as we would any other dictionary (or JSON) key in dot notation, with an index.
+
+**Indexes start at 0!** The first is not "1", we are dealing with code, not sheep.
+
+Let's take a look at the above context.
+
+Under "URLScan", we have an array called "Certificates". If we wanted to access the "SubjectName" value of the first entry, we would use the following DT statement.
+
+| Example | Result |
+| --- | --- |
+| `${URLScan.Certificates.SubjectName}` | \["www.google.com", "*.google.com", "www.google.de", "\*.g.doubleclick.net", "\*.apis.google.com"] |
+| ```${URLScan.Certificates.[0].SubjectName}``` | www.google.com |
+| `${URLScan.Certificates.[0:2].SubjectName}` | \["www.google.com", "*.google.com", "www.google.de"] |
+
+If you want to retrieve a range of results, you can use ```[0:9]``` where "0" is the beginning of the array and "9" is the 9th position in the array.
+
+## Selectors
+DT also allows for conditions within the statement itself and uses Javascript to select the context items. This functions as a way to bind results together by embedding a selection into the DT string.
+
+You will notice that `val` is used quite often in the DT string. This is a Javascript method that exposes the value at the given context path.
+
+The following are a few examples of selector methods:
+
+| Example | Result | Description |
+| --- | --- | --- |
+| ```${DBotScore.Vendor(val == 'urlscan.io')} ``` | `urlscan.io` | will return only Vendors that exactly match the Acrobat description. |
```

Comment:
I suspect a line in this table is too wide which is pushing the Table of Content to the bottom of the page. Suggest adding spaces in examples where appropriate to allow for word wrapping.